pdns/dynlistener.cc

```diff
@@ -214,7 +214,7 @@ string DynListener::getLine()
   vector<char> mesg;
   mesg.resize(1024000);
 
-  ssize_t len;
+  ssize_t len = 0;
```

Comment:
I'd prefer this declaration inside the last `if` block. Also, I think other tools will complain about the initialisation, claiming it is a dead assignment (which it is).